USAGE.md

```diff
@@ -62,6 +66,7 @@ await BackgroundService.stop();
 | `taskIcon`  | [`<taskIconOptions>`](#taskIconOptions) | **Android Required**. Notification icon. |
 | `color`  | `<string>` | Notification color. **Default**: `"#ffffff"`. |
 | `linkingURI`  | `<string>` | Link that will be called when the notification is clicked. Example: `"yourSchemeHere://chat/jane"`. See [Deep Linking](#deep-linking) for more info. **Default**: `undefined`. |
+| `actions`  | [`[<actionItem>]`](#actionItem) | **Android Required**. List of notification action items. |

```

Comment:
Is it Android Required? I think it might be better if it were an optional parameter so this feature could work as backwards compatible.

Reply:
Oh right, my bad. Optional definitely :)